css-view-transitions-1/Overview.bs

```diff
@@ -782,39 +785,39 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 	The initial phase is the first item in the sequence.
 
-## The snapshot root ## {#snapshot-root-concept}
+## The snapshot containing block ## {#snapshot-root-concept}
```

Comment:
```suggestion
## The snapshot containing block ## {#snapshot-containing-block-concept}
```

Reply:
Did you mean to resolve this after applying the suggestion? :)

Reply:
Oops, yes